src/Form/Checkbox.js

```diff
@@ -141,7 +145,7 @@ class Checkbox extends React.Component {
         <Flex alignItems="center">
           <StyledIcon size={iconSize} color={checked ? colorOn : colorOff} name={checked ? iconOn : iconOff} />
 
-          {label && <StyledLabel fontSize={fontSize}>{label}</StyledLabel>}
+          {label && <StyledLabel fontSize={fontSize} style={styles.Label}>{label}</StyledLabel>}
```

Comment:
This style is applied without specifying it in the styled component's template literal, I'm not sure why the other style does not.

Reply:
https://github.com/heydoctor/molekule/pull/28/files#r266171170